src/Seabug/CallContract.purs

```diff
@@ -71,12 +68,20 @@ import Serialization.Hash
   )
 import Types.BigNum as BigNum
 import Types.Natural as Nat
-import Wallet (mkNamiWalletAff)
+
+liftBuilder :: forall a. Either Error a -> Aff a
+liftBuilder = liftEffect <<< liftEither
```

Comment:
You may be able to just use `liftEither` by itself here, theres a `MonadThrow Error Aff` instance